guides/ecto.md

```diff
@@ -122,11 +125,11 @@ defmodule Hello.Repo do
 end
 ```
 
-It begins by configuring our `otp_app` name and repo module. Then it sets the adapter – Postgres, in our case. It also sets our login credentials. Of course, you can change these to match your actual credentials if they are different.
```

Comment:
I deleted the second sentence as nowhere in the code above it shows the credentials,
and it is more than obvious that can be edited in the code below. IMHO it adds no value.